classes/class-list-table.php

```diff
@@ -396,6 +396,12 @@ public function column_default( $item, $column_name ) {
 
 				if ( ! empty( $inserted_columns ) && is_array( $inserted_columns ) ) {
 					foreach ( $inserted_columns as $column_title ) {
+
+						// Set default value of column to column name.
+						if ( empty( $out ) ) {
+							$out = $column_name;
+						}
+
```

Comment:
As @derekherman stated above the loop has to be exited once the value is found or it will just overwrite itself with the default value.

@Nikschavan You can replace the whole conditional started on Line 397 with @derekherman suggestion above and that should work.